src/test/java/ch/liquidmind/inflection/compiler/InflectionCompilerTest.java

```diff
@@ -0,0 +1,55 @@
+package ch.liquidmind.inflection.compiler;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import ch.liquidmind.inflection.compiler.CompilationJob.CompilationMode;
+import ch.liquidmind.inflection.compiler.util.CompilerTestUtility;
+
+public class InflectionCompilerTest {
+
+	@Test
+	public void testCompile_ValidFile_SuccessfulCompilation() throws Exception {
+		CompilationJob job = CompilerTestUtility.createCompilationJob(CompilationMode.BOOTSTRAP, 
```

Comment:
Yes, yes, I know… all in good time ;-) The reserved names is the key point (as far as I remember).